src/ATen/native/xpu/sycl/Reduce.h

```diff
@@ -269,7 +269,10 @@ struct ReduceConfig {
     num_items = group_width * group_height;
 
     if (num_items < max_sg_sz)
+    {
       group_width = max_sg_sz;
+      num_items = group_width * group_height;
```

Comment:
Is it better to move line 269 behind the `if condition scope`.

Reply:
LGTM